content/guides/template-gds/_index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: 'GDS File Creation from Template'
+linkTitle: 'Create GDS Files With Our Github Template'
+description: 'The detailed procedure to get GDS from a wokwi template repo'
+weight: 60
+---
+
+
+This guide will help you through the essential steps to set up a GitHub repository, enable actions, and successfully commit your design for TinyTapeout. Follow each step carefully.
+
+## Use the Template Repository
+-------------------------------
+
+Start by using the template repository to create your own copy of the project. This will help you build upon an existing foundation.
+
+![Use Template Repository](images/template.png)
+
+## Enable GitHub Actions
+-------------------------
+
+Enabling GitHub Actions allows automated tasks to run, like testing or building your project. Make sure to activate actions for your repository.
+
+![Enable GitHub Actions](images/gitpages.png)
+
+## Update info.yaml
+--------------------
+
+The `info.yaml` file contains important configuration details about your project. Follow the subsections below to update this file properly.
+
+### a. Name and Description
+
+Update the `name` and `description` fields in the `info.yaml` file. These fields describe your project and give it a unique identity.
+
+    name: "Your Project Name"
+    description: "A brief description of your project"
+        
+
+### b. Set the Wokwi ID
+
+Add your Wokwi project ID in the `info.yaml` file under the `wokwi_id` field to connect your design to your Wokwi simulation.
+
+    wokwi\_id: '408122667678580737'
+        
+
+wokwi\_id is found from the number succeeding the URL in your WOKWI Project
+
+![Set the Wokwi ID](images/WOKWI_Project_ID.png)
+
+### c. Using More Tiles
+
+If your design needs more than one tile, update the `tiles` field in the `info.yaml` file. This allows you to expand the area for your design.
+
+![info.yaml](images/infoyaml.png) 
+![Commit the modified info.yaml](images/commitinfo.png)
+
+## Documentation
+-----------------
+
+Make sure to write clear documentation about your project for others to understand it. The `README.md` file is a good place to start.
```

Comment:
Add a note that the info.md documentation can't be left as the default or it will cause the docs action to fail